Framework/DataHandling/src/LoadTBL.cpp

```diff
@@ -320,24 +320,24 @@ void LoadTBL::exec() {
   if (isOld) {
     /**THIS IS ESSENTIALLY THE OLD LoadReflTBL CODE**/
     // create the column headings
+    auto colStitch = ws->addColumn("str", "StitchGroup");
     auto colRuns = ws->addColumn("str", "Run(s)");
     auto colTheta = ws->addColumn("str", "ThetaIn");
     auto colTrans = ws->addColumn("str", "TransRun(s)");
     auto colQmin = ws->addColumn("str", "Qmin");
     auto colQmax = ws->addColumn("str", "Qmax");
     auto colDqq = ws->addColumn("str", "dq/q");
     auto colScale = ws->addColumn("str", "Scale");
-    auto colStitch = ws->addColumn("int", "StitchGroup");
     auto colOptions = ws->addColumn("str", "Options");
 
+    colStitch->setPlotType(0);
     colRuns->setPlotType(0);
     colTheta->setPlotType(0);
     colTrans->setPlotType(0);
     colQmin->setPlotType(0);
     colQmax->setPlotType(0);
     colDqq->setPlotType(0);
     colScale->setPlotType(0);
-    colStitch->setPlotType(0);
     colOptions->setPlotType(0);
```

Comment:
If the aim here is to set PlotType as 0 for all the columns a loop would be neater and would prevent possible issues with future changes.